packages/react-examples/src/office-ui-fabric-react/DetailsList/DetailsList.GroupedV2.Example.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import {
+  DetailsHeader,
+  DetailsList,
+  IColumn,
+  IDetailsHeaderProps,
+  IDetailsList,
+  IGroup,
+  IRenderFunction,
+  IToggleStyles,
+  mergeStyles,
+  Toggle,
+  GroupedListV2_unstable as GroupedListV2,
+} from 'office-ui-fabric-react';
+import { DefaultButton, IButtonStyles } from 'office-ui-fabric-react/lib/Button';
```

Comment:
_nit_: You can pull all of these up into one import

```suggestion
import {
  DefaultButton,
  DetailsHeader,
  DetailsList,
  IButtonStyles,
  IColumn,
  IDetailsHeaderProps,
  IDetailsList,
  IGroup,
  IRenderFunction,
  IToggleStyles,
  mergeStyles,
  Toggle,
  GroupedListV2_unstable as GroupedListV2,
} from 'office-ui-fabric-react';
```